config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
 
   resources :posts
+  resources :advertisements
```

Comment:
yeah, you shouldn't expose routes you don't need. you've opened up a delete route for example, and you don't want people doing that.